data/connectors.js

```diff
@@ -0,0 +1,14 @@
+
+import Sequelize from 'sequelize'
+
+const db = new Sequelize('inspire', null, null, {
+  host: 'localhost',
+  dialect: 'postgres',
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000
```

Comment:
Please add a comment for the unit of this magic. I'd assume `ms` but I'd have to go look it up to know.